demos/SpatialCoop2017/source/web/SimplePDWorld-web.cpp

```diff
@@ -15,210 +20,145 @@ SimplePDWorld world;
 int cur_x = -1;
 int cur_y = -1;
 
+std::function<emp::SettingConfig()> SetupConfig = [](){
+    emp::SettingConfig config;
+    config.AddSetting<double>("r") = {world.GetR()};
+    config.AddSetting<double>("u") = {world.GetU()};
+    config.AddSetting<size_t>("N") = {world.GetN()};
+    config.AddSetting<size_t>("E") = {world.GetE()};
+
+    return config;
+};
+
+emp::SettingConfig config = SetupConfig();
+emp::QueueManager run_list(config);
+
 void DrawCanvas() {
-  UI::Canvas canvas = doc.Canvas("canvas");
-  canvas.Clear("black");
+    UI::Canvas canvas = doc.Canvas("canvas");
+    canvas.Clear("black");
 
-  const emp::vector<Org> & pop = world.GetPop();
+    const emp::vector<Org>& pop = world.GetPop();
 
-  if (cur_x >= 0) {
-    canvas.Circle(cur_x, cur_y, world_size*world.GetR(), "pink");
-  }
+    if (cur_x >= 0) {
+        canvas.Circle(cur_x, cur_y, world_size * world.GetR(), "pink");
+    }
 
```

Comment:
I'm going to skip over the source demos/ because it's not core source code (unless there's something specific I should look at/check out lmk)